build.sbt

```diff
@@ -42,7 +42,7 @@ val partestDep                   = scalaDep("org.scala-lang.modules", "scala-par
 // Non-Scala dependencies:
 val junitDep          = "junit"                  % "junit"                % "4.11"
 val junitInterfaceDep = "com.novocode"           % "junit-interface"      % "0.11"                            % "test"
-val scalacheckDep     = "org.scalacheck"         % "scalacheck_2.13.0-M1" % "1.13.5"                          % "test"
+val scalacheckDep     = "org.scalacheck"         % "scalacheck_2.13.0-M3" % "1.13.5"                          % "test"
```

Comment:
Since it's one of the things that is updated regularly, should we move it to `versions.properties`? With a comment why we're not using `scala.binary.version` (IIUC, it's because we're not building scalacheck, so we wait until it's available, and we don't want this to prevent us from moving `scala.binary.version` ahead).

Reply:
we already have `scala.binary.version` separate from `starr.version` for exactly that reason, so unless I'm missing something, we ought to just use `scala.binary.version` for ScalaCheck too

Reply:
The point is, I think, we update `scala.binary.version` immediately after a release together with `starr.version`. The reason to have them separates is to make the distinction before pre-`.0` (where `scala.binary.version` is exactly `starr.version`) and post-`.0` (where it's an actual binary version like `2.12`).

If we start using `scala.binary.version` for scalacheck, we have to wait updating it (in pre-`.0`) until scalacheck is out for a new milestone / RC / `.0`.

Reply:
okay, I'll revise